File: 백준/Silver/10814. 나이순 정렬/나이순 정렬.py
import sys

tc = int(sys.stdin.readline())
arr = []
for _ in range(tc):
    a, b = tuple(map(str, sys.stdin.readline().split()))
    a = int(a)
    arr.append((a, b))
    
arr.sort(key=lambda x: x[0])

for i in arr:
    print(i[0], i[1])

File: 백준/Silver/10814. 나이순 정렬/나이순 정렬.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int num = Integer.parseInt(br.readLine());
        String[][] arr = new String[num][2];
        for(int i = 0; i<num; i++){
            arr[i] = Arrays.stream(br.readLine().split(" ")).toArray(String[] :: new);
        }

        Arrays.sort(arr, Comparator.comparingInt(a -> Integer.parseInt(a[0])));

        for(String[] result : arr){
            System.out.println(result[0] + " " + result[1]);
        }

    }
}
